### ISSUE TITLE
Add stuff for next meeting

### ISSUE THREAD

**Issue by ghost** [alhanson7210/webPractice#1]
Hey. Let's add to this for the next meeting. 
The week after spring break should be DSA, and then we can follow up with this.

@alhanson7210 the goal of this is to teach them the basics of some front end.
Let's get together to see how we can make this a bit longer.

**Comment by alhanson7210** [comment]
I completely agree: Simple front end understanding is the aim of using this repo; with more time, comments and more documentation can be done to explain this further next week (after DSA).
